.travis.yml

```diff
@@ -31,3 +34,7 @@ notifications:
     on_failure: change
     rooms:
       secure: RfnO47KWRcZhzznK2zU7AQBQd8fmuXQFSCNQ0CpPM/4GywA6XBSVHbnI4uEr7lmEdB2WKktGHtKcfC0H/tCFkxBzovlkiHEsOPrc6/4RsJKrPYjO1qX35ttdiILkmClyo2V8cPOu2m5HhTYddn9wuKN/A7tjjNRHz8ani+CmVBnXMkbyRW5DlY1/q99Y4y2GLMhERyMqNZyeaGDoR4UOl2OuRExBZeN2E7gtUhf5LyCsD/995DQRHx0cJrAZh2tMcubdGOKH0hYVohUFjgeTBOu1Fcwn845i3cXp1IG5cx83EEsNSvcmux96EpMTlu+D7ABjFtUjTFXuDDjXhWGhdC3S3C5/UbVX4tNGbGq2ErSGS4TP8WRPthVyFnvOY2j4jWLmiDDUGyVY8rPsKd2PsO9JDMnL9GCJxwUlK222oAqdkObntm8bEjUk8YODgX7SsgeJ4s5Zp5nKnG/GBiWZAYp2WbiSMltEJiL/jTpAgUMfhVAefgHMi362TC97y/nYcUhGTftMqlx+ca3FlEayV0CTQd9CSjBKuZ/IGeE3qlZbZhWqQMt9gtvNaZ5yToAxNxfxIwJqgdaRlQ5ZJRJESrTXVRIo2v4+9qaofK57trz05oLREyTAUk/8hWGFTcOfXSQw/Up1ycGjtLSLFPuU3PJ7px4lQGfATonv1SgXfrg=
+addons:
+  sonarcloud:
+    token:
+      secure: HoD6oJ03Y60apDszjFZ8AN+ujnkbQAPM41bBb/YNNMtgslfnJPp63bQy5m0hoCsTVRoPiZvuZRpfxQpcJLmlEScY4ulQI7DRf+FFh6/2B80+kl1FRLsyar0/JtZvn/LDsP5GjksIZUD2cdPho2rRnmGiS9bmrKIc4YRKKxMO/zi33EUPgiwea23rCB/6/CK6AnAyD7ufwM/ldewqKiU5T67IbAQM2mKEMqyM+oPSx/HL6f9NrtpenZs9pP3y8oRCCty8yP5BOyoLqOJvn0lRqdAAL5S8Jsrb6NB7Mk+lrU4AcfcgEmDGbj+wKFdV/xlqZcCDGDvytqGocGsQZLwbaNlOzSTkoJVvzy1crMxySsqVFoGusISPXA3iJgBQn08E8bwSI1SWDfdvQTFZFYcoc/1OWNutH6EvoHC9uM9rmIRJPVDqGeBevlJq7vffspnuwpWZeH6hWQt7kHBGxQSYtCyOUkFU0IntRFiX2KvyP38rRIlfeoDM2fqaD/NUg0+C77YYhi9on74mQrw8EEjq5QErht458shfjiPPoq8PHpN2WNXjJ8lRHc6h8eqNzX0Uz2ooPOvxgFnyjergBbe5KADQkPLhJehZQhNtHiAfAX5dtkP8IJizu0tpwrTkMWrMMT51nbqL/zOtH6yflr1+AjKrLM7YFRSj8by3yokNzI4=
```

Comment:
[Q] Is this token generated from sonar?

Reply:
Used travis encryption for sonar cloud token 

```
travis encrypt --add addons.sonarcloud.token <SONAR_TOKEN>
```